lib/client/weblogin.go

```diff
@@ -653,11 +653,6 @@ func SSHAgentHeadlessLogin(ctx context.Context, login SSHLoginHeadless) (*authcl
 	}
 
 	re, err := clt.PostJSON(ctx, clt.Endpoint("webapi", "headless", "login"), req)
-	if trace.IsNotFound(err) {
-		// fallback to deprecated headless login endpoint
-		// TODO(Joerger): DELETE IN v18.0.0
-		re, err = clt.PostJSON(ctx, clt.Endpoint("webapi", "ssh", "certs"), req)
-	}
```

Comment:
It seems very scary to delete the server part in the same version as we delete the client part: with mixed versions I could see some fringe scenarios where the first POST fails because we hit an old proxy and the second one succeeds because we hit one that's too new. Can we delay the removal on the server side, or is it holding us back from making significant improvement in some areas?